### ISSUE TITLE
Shelly HomeKit 2.11.2 & Shelly 2.5 for roller shutter control. It stops working.

### ISSUE THREAD

**Issue by marmitol** [mongoose-os-apps/shelly-homekit#1231]
In my house with 4 Shelly 2.5 PM for roller shutter control.

Reference Version Shelly HomeKit 2.11.2. Bug.

All Shelly 2.5. Bug.
are displayed as "closed" in Homekit
are really "open"
don't react to the physical buttons
No action is possible anymore, neither via Homekit, nor via the wall buttons.

The only recovery I found so far is to start a new calibration, but for the ones that show this behavior it returns latest after a few days.

I have yet to look at the logs, because I only sporadically have access to the devices, but will add them here once I got them.

**Comment by marmitol** [comment]
I recently outfitted a friends house with 10 Shelly 2.5 PM for roller shutter control.
Two of them are currently showing the following behavior.
They

are displayed as "closed" in Homekit
are really "open"
don't react to the physical buttons
No action is possible anymore, neither via Homekit, nor via the wall buttons.

The only recovery I found so far is to start a new calibration, but for the ones that show this behavior it returns latest after a few days.

I have yet to look at the logs, because I only sporadically have access to the devices, but will add them here once I got them.
